setup/_metapackage/VERSION.txt

```diff
@@ -1 +1 @@
-16.0.20221111.0
\ No newline at end of file
+16.0.20221207.0
```

Comment:
And why this change?

Reply:
@pedrobaeza 
I don't know, I guess some process must create the setup directory.
How do you think it would be better to act: recover the previous value, or launch some command to correct it.

Reply:
Rebase and remove this change

Reply:
This is still happening.